src/main/java/com/welovecoding/nbeditorconfig/config/Settings.java

```diff
@@ -5,6 +5,7 @@
 public final class Settings {
 
   public static final String MIME_TYPE = "text/x-editorconfig"; // NOI18N
+  public static final String EXTENSION = "editorconfig"; // NOI18N
```

Comment:
Always keep it simple. There is **NO** automatic text translation right now and it makes **NO** sense to translate a MIME type. The documentation says you should use `NOI18N` when it is **not completely obvious** that something should not be translated and in this case it is obvious. So it's **pointless** to use `NOI18N` here.